genesis/ext/pyrender/viewer.py

```diff
@@ -14,15 +14,8 @@
 
 import genesis as gs
 
-try:
-    from Tkinter import Tk
-    from Tkinter import tkFileDialog as filedialog
-except Exception:
-    try:
-        from tkinter import Tk
-        from tkinter import filedialog as filedialog
-    except Exception:
-        pass
```

Comment:
The purpose of this code is to ensure compatibility with Tkinter, supporting both Python 2 and Python 3. Genesis requires Python 3.9 or higher, such compatibility adjustments are unnecessary.

https://github.com/Genesis-Embodied-AI/Genesis/blob/2985aa9c63b4162cea10a52c98161e1270477746/pyproject.toml#L12